### DISCUSSION TITLE
v5.0.1_33_BETA

### DISCUSSION THREAD

**Discussion by Alexco500** [Alexco500/sidplay5#14] [General]
This is a beta release, it should fix the following issues:
- close window gadget now works like other macOS apps
- Dark Mode should work now
- Audio should now stream correctly with AirPods, HomePods, ....

Since this is a BETA, it is not notarized.
And now back to Revision party 2023...

<hr /><em>This discussion was created from the release <a href='https://github.com/Alexco500/sidplay5/releases/tag/v5.0.1_33_BETA'>v5.0.1_33_BETA</a>.</em>

**Comment by Brog33** [comment]
Hey, hope you had a blast at Revision. Can I try this beta?

EDIT: Just found it!

**Reply by Brog33** [reply]
Dark mode looks great! Good work.

Close button: Closes the window, but keeps the SID playing in the background. When I click the app icon in the dock, it won't open the app window again, so I have to quit and restart the app to get the app window back.

**Reply by Alexco500** [reply]
There should be a menu item in the window menu which will show the main window again.
But clicking on the dock icon should also be possible. I will simply add that.
